tests/__init__.py

```diff
@@ -30,14 +30,12 @@ class BaseTest(BaseTestCase):
         'description': 'base lorem ipsum - english',
         'og_description': 'opengraph - lorem ipsum - english',
         'twitter_description': 'twitter - lorem ipsum - english',
-        'gplus_description': 'gplus - lorem ipsum - english',
```

Comment:
```
System check identified no issues (0 silenced).
.......E.............
======================================================================
ERROR: test_page_meta_no_meta (tests.test_general.PageMetaUtilsTest)
Tests the meta if no PageMeta is set
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./tests/test_general.py", line 88, in test_page_meta_no_meta
    self.assertEqual(meta.published_time, page.publication_date.isoformat())
AttributeError: 'Meta' object has no attribute 'published_time'
```

I removed `test_page_meta_no_meta` because it failed (since I removed gplus content).